File: matiasmasso/Mat.NET/DTO/DTOECITransmCentre.cs
using System;

namespace DTO
{
    public class DTOECITransmCentre : DTOBaseGuid
    {
        public DTOECITransmGroup Parent { get; set; }

        public DTOContact Centre { get; set; }

        public DTOECITransmCentre() : base()
        {
        }

        public DTOECITransmCentre(Guid oGuid) : base(oGuid)
        {
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOECITransmCentre.vb
Public Class DTOECITransmCentre
    Inherits DTOBaseGuid

    Property Parent As DTOECITransmGroup

    Property Centre As DTOContact

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class
